project/Compiling.scala

```diff
@@ -14,6 +14,6 @@ object Compiling {
   }
 
   val rawlsCompileSettings = List(
-    resourceGenerators in Compile <+= writeVersionConf
+    resourceGenerators in Compile += {writeVersionConf}
```

Comment:
When I tried this upgrade (and didn't really know what I was doing) I came up with:
```
Compile / resourceGenerators += writeVersionConf
```
Not even sure if that works or which is better. 🤷‍♂️